cmd/pebble/main.go

```diff
@@ -348,8 +344,15 @@ func run() error {
 		return fmt.Errorf("cannot create client: %v", err)
 	}
 
+	var xtra []string
 	parser := Parser(cli)
-	xtra, err := parser.Parse()
+	if err2 := boot.CheckBootstrap(); err2 == nil {
```

Comment:
I didn't understand this at first, then I saw in the spec we want `boot-firmware` to be the default if no command is specified.

To help clarify, perhaps we can make this something other than `CheckBootstrap`. Perhaps `boot.IsTermus() bool` that just checks for the environment variable (or just check for the environment variable directly in main)?

Reply:
I don't see the need to duplicate the bootstrap checks here: these checks are not expensive in terms of CPU time and also we should not do partial checks (i.e. checking *only* the environment variable) because we could be in a scenario in which a user invokes Pebble from a shell (and the `TERMUS` env var will be `1` since it's inherited down in the process tree), thus triggering the unwanted behavior of Pebble running the `boot-firmware` command by default.

I do agree that we should name it something more appropriate. Thanks for your feedback!

Reply:
On second thought, after replacing all build tag boilerplate from `internal/boot` (now `internal/bootstrap`), it has become impossible to continue with this approach (because these symbols simply do not exist on the non-`termus` variant of Pebble). I've implemented your solution and would love to hear your opinion (: